src/ai/backend/manager/exceptions.py

```diff
@@ -25,6 +25,29 @@ class InvalidArgument(Exception):
     pass
 
 
+class RPCError(RuntimeError):
+    """
+    An exception class to represent any error caused in RPC functions.
+    """
+
+    __slots__ = (
+        "agent_id",
+        "agent_addr",
+        "exc_repr",
+    )
+
+    def __init__(
+        self,
+        agent_id: AgentId,
+        agent_addr: str,
+        exc_repr: str,
+    ) -> None:
+        super().__init__(agent_id, agent_addr, exc_repr)
+        self.agent_id = agent_id
+        self.agent_addr = agent_addr
+        self.exc_repr = exc_repr
```

Comment:
To match convention I think this variable is better to be named as `extra_msg`.
```suggestion
        self.extra_msg = extra_msg
```